miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSubmissionController.java

```diff
@@ -79,13 +79,13 @@ public ModelAndView newSubmission(@QueryParam("experimentIds") String experiment
   }
 
   @RequestMapping(method = RequestMethod.POST)
-  public String processSubmit(@ModelAttribute("submission") Submission submission, ModelMap model, SessionStatus session)
+  public ModelAndView processSubmit(@ModelAttribute("submission") Submission submission, ModelMap model, SessionStatus session)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Replace this persistent entity with a simple POJO or DTO object. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4684)